questions.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Ask the Candidates a Question!
+---
+
+# {{ page.title }}
+
+The .NET Foundation's Board of Directors will be working with you and for you, and so we want to make sure you have the ability to ask us any questions you might have about our plans or our approach to the Foundation and our potential seat on the board.
+
+## Asking a Question
+
+Questions are being asked [on GitHub](https://github.com/dotnet-foundation/election/labels/candidate-question).
+
+To submit a question, [create a new issue](https://github.com/dotnet-foundation/election/issues/new) and use the "Candidate Question" template. We'll label the question once it's submitted.
```

Comment:
We can improve this link using [GitHub query params](https://help.github.com/en/articles/about-automation-for-issues-and-pull-requests-with-query-parameters#supported-query-parameters). Something like https://github.com/dotnet-foundation/election/issues/new?title=Election+Question&labels=candidate-question&template=electionquestion.md (won't totally work yet because the label and issue template don't exist).

If we really wanted to get fancy we could provide text entry boxes right on the page and then auto-format and populate the body of the issue using JS (that's what I do here https://discoverdot.net/suggest/project) - think there's value in that or just leave it at a link?

Reply:
@daveaglick good catch on the query params! I'll add it.

Re: text boxes & formatting, I think that's a bit much at this point but wouldn't mind a follow-up PR after we get this up there.

Reply:
Thanks for being awesome